Documentation/operations/upgrade.rst

```diff
@@ -310,240 +310,15 @@ Annotations:
   add an additional ``--bpf-lb-external-control-plane=true`` to the command line, otherwise
   it is assumed that Kubernetes is present.
 
-.. _1.16_upgrade_notes:
-
-1.16 Upgrade Notes
```

Comment:
Could you update this to 1.17? with the same section headers as before?

Usually contributors need a hook of where to contribute their upgrade notes, so it helps to keep the structure alive early in the dev cycle.

Reply:
1.17 was above it. I still added other sections though for easier development.